packages/core/src/browser/style/variables-bright.useable.css

```diff
@@ -148,8 +148,9 @@ is not optimized for dense, information rich UIs.
   --theia-removed-color0: rgba(230, 0, 0, 0.8);
   --theia-modified-color0: rgba(0, 100, 150, 0.8);
 
-  --theia-highlight-background-color: var(--md-purple-A100);
-  --theia-highlight-color: var(--theia-content-font-color0);
+  --theia-search-match-color0: rgba(234, 92, 0, 0.33);
+  --theia-search-match-color1: rgba(234, 92, 0, 0.5);
+  --theia-match-replace-color: rgba(155, 185, 85, 0.2);
```

Comment:
Why cannot we leave here if the followings are here?
```css
  --theia-added-color0: rgba(0, 255, 0, 0.8);
  --theia-removed-color0: rgba(230, 0, 0, 0.8);
  --theia-modified-color0: rgba(0, 100, 150, 0.8);
```

Why would search belong to the tree?

Reply:
Good point. They should be moved too. Into here: https://github.com/theia-ide/theia/blob/master/packages/git/src/browser/style/index.css#L7 or directly to https://github.com/theia-ide/theia/blob/master/packages/git/src/browser/style/dirty-diff.css#L9 (the only place they are actually used for now). I am not sure about that.

Regarding the tree.css. I've associated the variables to it because it is about markers in trees. 
But I guess you are right. Doesn't feel good there. So we need a search.css or such. 
The important thing with such variables is also that they have the same values in every theme. We don't have a mechanism to switch in other files than the variables...useable.css files. 
I guess there is still potential for improvements regarding the theming.

Reply:
But if I think about it. We don't want to have other css files to be switched between themes. There should be just those too theming files. But with more basic variables with a not so specialised naming I guess.

Reply:
What about using the following names:
`--theia-search-match-color0:` -> `--theia-word-highlight-match-color0`
`--theia-search-match-color1:` -> `--theia-word-highlight-match-color1`
`--theia-match-replace-color:` -> `--theia-word-highlight-replace-color0`

We already have the `-theia-word-highlight-color0` and `-theia-word-highlight-color1` variables.